hhs_oauth_server/settings/base.py

```diff
@@ -117,15 +117,10 @@
 
 ROOT_URLCONF = 'hhs_oauth_server.urls'
 
-# Place all environment/installation specific code in a separate app
-# hhs_oauth_server.hhs_oauth_server_context.py also
-# includes IsAppInstalled to check for target_app in INSTALLED_APPS
-# This enables implementation specific code to be branched inside views and
-# functions.
 TEMPLATES = [
```

Comment:
This matches the old Ansible version of this variable, which is slightly different. I don't believe this is functional, but this is what we were using on sandbox.